### ISSUE TITLE
Using Pytorch schedulers 

### ISSUE THREAD

**Issue by Deepblue129** [IBM/pytorch-seq2seq#37]
Instead of updating the learning rate @ Optim.py. There are Pytorch schedulers that are supported.

http://pytorch.org/docs/master/optim.html#how-to-adjust-learning-rate

**Comment by kylegao91** [comment]
Thanks, didn't notice they have that.
